### PR TITLE
Feature/remove not used file

### PR DESCRIPTION
**Pull request by 9547**
### What problem does this PR solve? 

[pkg/cluster/template/config/dm/prometheus.go](pkg/cluster/template/config/dm/prometheus.go) is not used anymore, remove it

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/flags/debug.go [hunk 1]
```diff
@@ -24,7 +24,6 @@ var (
 )
 
 func init() {
-	if s := os.Getenv("TIUP_CLUSTER_DEBUG"); s == "enable" {
-		DebugMode = true
-	}
+	ShowBacktrace = len(os.Getenv("TIUP_BACKTRACE")) > 0
```
Comment (lucklove) [inline pkg/cluster/flags/debug.go:27]: For the env which can be modify by user, better to define it in [localdata](https://github.com/pingcap/tiup/blob/master/pkg/localdata/constant.go#L26) and list it in [envList](https://github.com/pingcap/tiup/blob/master/cmd/env.go#L24)

File: pkg/cluster/flags/debug.go [hunk 2]
```diff
@@ -24,7 +24,6 @@ var (
...
+	DebugMode = os.Getenv("TIUP_CLUSTER_DEBUG") == "enable"
```
Comment (lucklove) [inline pkg/cluster/flags/debug.go:28]: ditto

File: pkg/cluster/flags/debug.go (hunk 1 above)
Comment (9547) [reply pkg/cluster/flags/debug.go:27]: I got it

Comment (9547) [reply pkg/cluster/flags/debug.go:27]: BTW, `ShowBacktrace` was not used, I'll remove it.

Comment (9547) [reply pkg/cluster/flags/debug.go]: one more thing, `tiup env` currently only displays the env with  `key=val`, should we extend it with `key=val description`, the description will tell users how to set the env, or the usage of this env?

Comment (lucklove) [reply pkg/cluster/flags/debug.go]: Maybe the  current style (same with go env) is more friendly for script?

**Pull request merged**